### DISCUSSION TITLE
ENV variable for Airflow Metrics

### DISCUSSION THREAD

**Discussion by exilemirror** [apache/airflow#17713] [Q&A] [answered]
Hi guys, 

I'm running airflow 2.1.2 in docker. In my docker-compose env variables, I have define the metrics setup as below:
AIRFLOW__METRICS__STATSD_ON: 'true'
AIRFLOW__METRICS__STATSD_HOST: 'datadog'

The above metrics value were not updated in airflow.cfg inside the container.
How can I update the metrics config using env variables?

Thank you.

**Comment by waiyan1612** [comment, marked as answer]
I don't think it is going to update `airflow.cfg` in the container. You can check https://{{your-airflow-web-server}}/configuration to see if the configs are being picked up from cfg file or from the env var.

<img width="1653" alt="config-ui" src="https://user-images.githubusercontent.com/8967715/130022662-d8b90d68-66d3-4dc6-9b3b-5a4fce962f45.png">

**Reply by exilemirror** [reply]
Thanks for the reply @waiyan1612 
I tested it and checked the configurations on the GUI, yes you are right it doesn't overwrite the airflow.cfg file
